src/app/components/room/room.service.js

```diff
@@ -144,12 +144,21 @@
             ActionService.enterRoom(msg.id, username, connection);
             // Step 3. Establish WebRTC connection with the Janus server
             // Step 4a (parallel with 4b). Publish our feed on server
+
+            var participants = 0;
             connection.publish({
+              success: function() {
+                  //start session unmuted?
+                  if(participants < 4){
```

Comment:
I had that thought but connection.publish is an async network operation and will never fire that fast, so I wanted to keep the code short.